docs/en/latest/apisix.md

```diff
@@ -31,6 +31,14 @@ helm repo update
 helm install apisix apisix/apisix --create-namespace  --namespace apisix
 ```
 
+### Start By One Line
+
+To quickly experience Apache APISIX related components like Apache APISIX Dashboard and Apache APISIX Ingress Controller. For a quick installation, start by one line:
+
+```shell
```

Comment:
we should add `apisix` and `etcd` repo, first.

Reply:
The `etcd` repo is automatically dependent, adding `apisix` repo is enough. I have fixed it. PTAL.